sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md

```diff
@@ -0,0 +1,104 @@
+## Azure Key Vault Secrets Spring boot starter client library for Java
+Azure Key Vault Secrets Spring boot starter is Spring starter for [Azure Key Vault Secrets](https://docs.microsoft.com/rest/api/keyvault/about-keys--secrets-and-certificates#BKMK_WorkingWithSecrets). With this starter, Azure Key Vault is added as one of Spring PropertySource, so secrets stored in Azure Key Vault could be easily used and conveniently accessed like other externalized configuration property, e.g. properties in files.
+
+## Key concepts
+
+## Getting started
+### Add the dependency
+
+`azure-spring-boot-starter-keyvault-secrets` is published on Maven Central Repository.  
+If you are using Maven, add the following dependency.  
+
+[//]: # ({x-version-update-start;com.azure:azure-spring-boot-starter-keyvault-secrets;dependency})
```

Comment:
This should be current not dependency. The version of this gets updated when you set the version just prior to release.

Reply:
changed.